### PR TITLE
0.9.0 release prep

### LINKED ISSUE #1206: Prep for 0.9 Release
- [ ] Update changelog

### PR DESCRIPTION
**Pull request by Shoeboxam**
Closes #1206

### CODE REVIEW

**Comment by Shoeboxam** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #1207** <a href="https://app.graphite.dev/github/pr/opendp/opendp/1207?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/opendp/opendp/1207?utm_source=stack-comment).

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ showing the source changes from the previous version.
...
+- The OpenDP Python package now supports PEP 561 type information [#738](https://github.com/opendp/opendp/pull/738)
+- The OpenDP Rust crate is now thread-safe [#874](https://github.com/opendp/opendp/pull/874)
+- Documentation, Typing and CI improvements from Chuck McCallum
+    - CI: MyPy type-checking, link-checking in docs, code coverage, Rust formatting
+    - Rust stack traces are now hidden by default
+- FFI module in Rust is now public, allowing you to write your own lightweight FFI
+- C dependencies on GMP/MPFR have been replaced with dashu
+    - The OpenDP Rust library can now be built easily on Windows and is a much more lightweight Rust dependency
+
+# Changed
```
Comment (mccalluc) [inline CHANGELOG.md:36]:
```suggestion
### Changed
```

**Review by mccalluc — CHANGES_REQUESTED** [review]
Tiny fix to the markdown, otherwise approved.

Background questions:
- Would it be worth adding PRs links for dashu and ffi? Don't want to invent busy work, but curious why some have links and some don't.
- Have you considered building up the CHANGELOG incrementally in an `Unrealeased` section? Although that might be awkward with the PR stacks, and there might be other reasons that wouldn't work here.

**Comment by Shoeboxam** [comment]
The 0.9.0-dev section is where unreleased content would go. We could start maintaining it as we go, but I think it will cause more conflicts and probably need to be revised anyways before release. Regardless, going forward, lets try to maintain it as we merge into main and see how it goes.

**Review by mccalluc — APPROVED** [review]
Approved!

**Pull request merged**